Sources/FirebaseFirestore/Firestore+Swift.swift

```diff
@@ -34,6 +34,63 @@ extension Firestore {
   public func collection(_ collectionPath: String) -> CollectionReference {
     swift_firebase.swift_cxx_shims.firebase.firestore.firestore_collection(self, std.string(collectionPath))
   }
+
+  public func runTransaction(_ updateBlock: @escaping (Transaction, UnsafePointer<NSError?>?) -> Any?, completion: @escaping (Any?, Error?) -> Void) {
```

Comment:
Will this need changes from https://github.com/thebrowsercompany/swift-firebase/pull/30/files for the error pointer?

Reply:
Yes, that PR builds on top of this one. I wrote that one once I was able to try building against Arc. Rather than complicate things by folding it into this PR, I just left them as separate. Hope that works!